src/main/scala/com/fulcrumgenomics/bam/FilterBam.scala

```diff
@@ -62,7 +68,11 @@ class FilterBam
   @arg(flag='D', doc="If true remove all reads that are marked as duplicates.")   val removeDuplicates: Boolean = true,
   @arg(flag='U', doc="Remove all unmapped reads.")                                val removeUnmappedReads: Boolean = true,
   @arg(flag='M', doc="Remove all mapped reads with MAPQ lower than this number.") val minMapQ: Int = 1,
-  @arg(flag='S', doc="Remove all reads marked as secondary alignments.")          val removeSecondaryAlignments: Boolean = true
+  @arg(flag='P', doc="Remove all reads that are not part of a mapped pair.")      val removeUnpairedReads: Boolean = false,
+  @arg(flag='S', doc="Remove all reads marked as secondary alignments.")          val removeSecondaryAlignments: Boolean = true,
+  @arg(          doc="Remove all reads with insert size < this value.")           val minInsertSize: Option[Int] = None,
```

Comment:
I would prefer inclusive, say `<=` rather than `<`.  Similarly for `maxInsertSize`.  I read it as "all reads with at least insert size of..."